docs/ide/add-interface-definition-library-method-wizard.md

```diff
@@ -42,7 +42,7 @@ The following section describes the wizard interface that you'll use to add a me
 
 - **Return type**
 
-  The data type returned by the method. The standard way to return error codes from methods defined in an interface is with a `HRESULT`.
+  The data type returned by the method. The standard way to return error codes from methods defined in an interface is with an `HRESULT`.
```

Comment:
should be 'a' not 'an'  It's a weird case but based on the pronunciation it should be 'a'